language/move-vm/runtime/src/loader.rs

```diff
@@ -561,29 +563,60 @@ impl Loader {
     // A module to be published must be loadable.
     // This step performs all verification steps to load the module without loading it.
     // The module is not added to the code cache. It is simply published to the data cache.
-    // See `verify_script()` for script verification steps.
-    pub(crate) fn verify_module_verify_no_missing_dependencies(
+    // (See `verify_script()` for script verification steps(
```

Comment:
Seems like this comment should end with a close paren

Reply:
parser people....